packages/code/src/Code/Code.styles.ts

```diff
@@ -112,6 +154,47 @@ export function getCodeWrapperVariantStyle(theme: Theme): string {
   `;
 }
 
+export const expandButtonStyle = css`
+  align-items: center;
+  border: none;
+  /**
+    Code wrapper's border radius is 12px. Matching that creates a very 
+    slight gap between the button and the code wrapper. Decreasing by
+    1px removes gap without making focused state look off.
+  */
+  border-radius: 0 0 11px 11px;
+  border-width: 1px 0 0 0;
+  border-style: solid;
+  display: flex;
+  font-family: ${fontFamilies.default};
+  font-size: ${typeScales.body1.fontSize}px;
+  gap: 4px;
+  grid-area: expandButton;
+  justify-content: center;
+  z-index: 2;
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+export function getExpandButtonVariantStyle(theme: Theme): string {
+  const colors = variantColors[theme];
+
+  return css`
+    background-color: ${colors[0]};
+    border-color: ${colors[1]};
+    color: ${colors[2]};
+    &:hover {
+      background-color: ${colors[1]};
+    }
+    &:focus {
+      background-color: ${
```

Comment:
Would any of our color tokens work here? https://github.com/mongodb/leafygreen-ui/blob/main/packages/tokens/src/color/color.ts

Reply:
I was able to find one for the background, but I'm not sure I'm seeing one for the color. Feel free to let me know if I'm missing that though.